force-app/main/service/AddressService.cls

```diff
@@ -63,6 +63,10 @@ public inherited sharing class AddressService {
         sobjDst.put(strFieldPrefixDst + 'Country', sobjSrc.get(strFieldPrefixSrc + 'Country'));
         sobjDst.put(strFieldPrefixDst + 'Latitude', sobjSrc.get(strFieldPrefixSrc + 'Latitude'));
         sobjDst.put(strFieldPrefixDst + 'Longitude', sobjSrc.get(strFieldPrefixSrc + 'Longitude'));
+        if (orgConfig.isStateCountryPicklistsEnabled()) {
+            sobjDst.put(strFieldPrefixDst + 'StateCode', sobjSrc.get(strFieldPrefixSrc + 'StateCode'));
```

Comment:
Do we still want to do this after the code reversion???

Reply:
Yes, this method is for Contact <-> Account and Vice Versa, and we need as much information as possible when we eventually create NPSP Addresses.